hosted_control_planes/hcp-getting-started.adoc

```diff
@@ -33,7 +33,7 @@ You can view the procedures by selecting from one of the following providers:
 == {VirtProductName}
 
 * link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#hosted-sizing-guidance[Hosted control plane sizing guidance]
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#hosted-install-cli[Installing the hosted control plane command line interface]
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#hosted-install-cli[Installing the hosted control plane command-line interface]
```

Comment:
This file has the following changes: 
- command line interface --> command-line interface
- non bare metal --> non-bare-metal (ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words and SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_b)